tasks/eth/base-002-fp-upgrade/.env

```
@@ -0,0 +1,8 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0x95703e0982140d16f8eba6d158fccede42f04a4c
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+# This is Base's safe but the Nested.just file uses council as a keyword and
+# we want to minimize changes
+COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1
+FOUNDATION_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A
```

I think that make more sense if we use use the _Foundation Upgrade Safe_ (FuS) instead of the _Foundation Operation Safe_ (FoS). 
By replacing the address to `0x847B5c174615B1B7fDF770882256e2D3E95b9D9` for standard reason. 
Even if the nonce of the FnD safe are not going to be increased as this is nested and not executed from FnD.  


 
```suggestion
FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D9
```

---

The COUNCIL_SAFE and FOUNDATION_SAFE here are the two signers on the OWNER_SAFE. Unfortunately, to your point, the foundation signer is still 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A and therefore burns 1 presigned pause, but for the sake of this playbook we must leave it as is

I have created https://github.com/ethereum-optimism/security-pod/issues/183 for us to track this